core/src/test/scala/integration/kafka/api/AdminClientTest.scala

```diff
@@ -106,7 +106,7 @@ class AdminClientTest extends IntegrationTestHarness with Logging {
 
     val consumerSummaries = client.describeConsumerGroup(groupId)
     assertEquals(1, consumerSummaries.size)
-    assertEquals(Set(tp, tp2), consumerSummaries.head.assignment.toSet)
+    assertEquals(Set(tp, tp2), consumerSummaries.get.head.assignment.toSet)
```

Comment:
It's a bit nicer to do:

```scala
assertEquals(Some(Set(tp, tp2)), consumerSummaries.map(_.head.assignment.toSet))
```

The error message in case `None` is returned will be nicer if you do it like the above.